src/app/producer-shelf/components/tv2-action-card/tv2-action-card.component.ts

```diff
@@ -86,6 +88,15 @@ export class Tv2ActionCardComponent implements OnInit, OnDestroy {
     return this.getActionContentType()
   }
 
+  @HostBinding('class.missing-source')
+  public get showMissingMediaSource(): boolean {
+    return this.shouldShowMediaUnavailable()
+  }
+
+  private shouldShowMediaUnavailable(): boolean {
+    return this.isVideoClipAction() && !this.media
+  }
```

Comment:
`showMissingMediaSource` sounds like an action to perform. It would be nice if the name exhibited that it is a boolean query. A suggestion could be to merge these two methods into the getter that the host binding uses.

Reply:
Updated